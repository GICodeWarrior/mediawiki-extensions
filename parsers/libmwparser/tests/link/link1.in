[[Link|text''format''text

{|
| column1
|}
[[
test
]]
