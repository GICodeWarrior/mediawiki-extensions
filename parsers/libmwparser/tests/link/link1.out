BEGIN ARTICLE
    BEGIN PARAGRAPH
        SPECIAL[[]
        SPECIAL[[]
        WORD[Link]
        SPECIAL[|]
        WORD[text]
        BEGIN ITALIC
            WORD[format]
        END ITALIC
        WORD[text]
    END PARAGRAPH
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[column]
                    WORD[1]
                    NEWLINE
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN PARAGRAPH
        SPECIAL[[[]
        NEWLINE
        WORD[test]
        NEWLINE
        SPECIAL[]]
        SPECIAL[]]
    END PARAGRAPH
END ARTICLE
